com.palantir.typescript/src/com/palantir/typescript/IPreferenceConstants.java

```diff
@@ -41,6 +41,7 @@
     String COMPILER_OUT_FILE = "compiler.outputFileOption";
     String COMPILER_REMOVE_COMMENTS = "compiler.removeComments";
     String COMPILER_SUPPRESS_IMPLICIT_ANY_INDEX_ERRORS = "compiler.suppressImplicitAnyIndexErrors";
+    String COMPILER_SUPPRESS_EXCESS_PROPERTY_ERRORS = "compiler.suppressExcessPropertyErrors";
```

Comment:
Ah, I only thought to group them as far as "suppress". No problem, one sec

On Wed, Sep 30, 2015 at 3:35 PM, Derek Cicerone <notifications@github.com>
wrote:

> In
> com.palantir.typescript/src/com/palantir/typescript/IPreferenceConstants.java
> <https://github.com/palantir/eclipse-typescript/pull/282#discussion_r40840552>
> :
>
> > @@ -41,6 +41,7 @@
> >      String COMPILER_OUT_FILE = "compiler.outputFileOption";
> >      String COMPILER_REMOVE_COMMENTS = "compiler.removeComments";
> >      String COMPILER_SUPPRESS_IMPLICIT_ANY_INDEX_ERRORS = "compiler.suppressImplicitAnyIndexErrors";
> > +    String COMPILER_SUPPRESS_EXCESS_PROPERTY_ERRORS = "compiler.suppressExcessPropertyErrors";
>
> very cool - can i ask for one small change across the patch? i typically
> sort things so could you put the new property above
> suppressImplicitAnyIndexErrors all over?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/eclipse-typescript/pull/282/files#r40840552>.
>